scripts/cache-load.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# If the environment variable S3_BUILD_CACHE is set, download and extract the
+# tarball it points to into the directory $HOME/.s3buildcache.
+#
+set -eu
+
+cachedir=$HOME/.s3buildcache
+mkdir -p $cachedir
```

Comment:
`~` (Nit) - Move this to after the env variable check so this script no-ops if the environment variable isn't set.

Reply:
Actually I did this on purpose. It's convenient to always be able to rely on the `~/.s3buildcache` directory existing (whether it's empty or not).

Makes the shell scripts easier to write... (?)